corehq/apps/cloudcare/static/cloudcare/js/utils.js

```diff
@@ -359,13 +359,13 @@ hqDefine('cloudcare/js/utils', [
     var convertTwoDigitYear = function (inputDate) {
         var parts = inputDate.split(/\D/);
         if (parts.length === 3 && parts.join("").length <= 6) {
-            var inputYear = parts[2];
-            if (inputYear.length === 2) {
-                inputYear = Math.floor(new Date().getFullYear() / 100) + inputYear;
-                if (inputYear > new Date().getFullYear() + 10) {
-                    inputYear -= 100;
+            let [day, month, year] = parts;
```

Comment:
nit: I do like the naming for clarity, but hesitant with the `day` and `month` variable names because in the context of `convertTwoDigitYear`,` part[0] `could be either day or month and same with `part[1]`

Reply:
(:facepalm: I got them backwards - just switched it)

That's true - this function will work with either, since it leaves month and day alone, but the dates returned from this function _are_ going to be interpreted as if they are month/day/year.  I don't love that this is coupled so closely to the date format, since the format is ostensibly variable (I could see us making the default format configurable per project, for example).  It would be nice to just override `moment.parseTwoDigitYear` like you suggested, though I don't want to affect other places that's used.